dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/PortRangeBean.scala

```diff
@@ -0,0 +1,8 @@
+package com.lightbend.lagom.maven
+
+import scala.beans.BeanProperty
+
+class PortRangeBean {
```

Comment:
This is going to be a singleton, or am I wrong? I'm wondering because I'd expect the min/max may be write/read from different threads (and, if so, we will need to establish some sort of memory barrier to avoid race conditions or visibility issues).